### DISCUSSION TITLE
IRHVAC SAMSUNG_AC how to send "Breeze" ON/OFF according to latest IRremoteESP8266?

### DISCUSSION THREAD

**Discussion by andreagiaquinto** [arendst/Tasmota#18824] [Q&A] [answered]
Hello,
there is no option "Breeze" ON/OFF to send using IRHVAC.
Is not integrated yet in tasmota_ir?

Thank you.

Kind Regards
Andrea

**Comment by daniele-athome** [comment]
ASAIU, the library Tasmota uses for IRHVAC did not expose the Breeze methods in the API common to all HVAC models: https://github.com/crankyoldgit/IRremoteESP8266/issues/1062

I don't know if it would be better to hack into IRremoteESP8266 to expose that as a common option (although it's pretty specific to Samsung) or put some Samsung-specific code into Tasmota.

**Comment by daniele-athome** [comment]
For anyone coming here in the future: I ended up [forking Tasmota and making a little patch](https://github.com/daniele-athome/Tasmota/commit/development). It works for my needs (fan speed auto == breeze is fine for me) and maybe whoever else has a Samsung Windfree A/C :-)

**Reply by s-hadinger** [reply]
This needs to be submitted as a PR to IRRemoteESP2866 (if not already done)

**Reply by daniele-athome** [reply, marked as answer]
I'm afraid it can't because of [a couple of reasonable reasons](https://github.com/crankyoldgit/IRremoteESP8266/issues/1062#issuecomment-2174627305). It is a brand-specific feature that can't really be generalized.

**Comment by bluewalk** [comment]
I am coming across this discussion trying to send the setBreeze command to my AC. Only to find out it is not supported.
Is it possible to have brand specific settings to be sent via the IRHVAC command as we are specifying the vendor `SAMSUNG_AC`?

If not, I do see the `iFeel` being reported, even though this appears to be also brand specific: Gree A/C and Argo Ulisse).
So why is this supported? Can we somehow have the `iFeel` value to be used to set the `breeze` value for Samsung AC units?

```json
{
    "Vendor": "SAMSUNG_AC",
    "Model": -1,
    "Command": "Control",
    "Mode": "Heat",
    "Power": "Off",
    "Celsius": "On",
    "Temp": 27,
    "FanSpeed": "Auto",
    "SwingV": "Off",
    "SwingH": "Off",
    "Quiet": "Off",
    "Turbo": "Off",
    "Econo": "Off",
    "Light": "On",
    "Filter": "Off",
    "Clean": "Off",
    "Beep": "Off",
    "Sleep": -1,
    "iFeel": "Off",
    "SensorTemp": null
}
```

**Reply by sfromis** [reply]
Expanded device support is not handled in Tasmota, but instead in:
https://github.com/crankyoldgit/IRremoteESP8266/
Do read the post right above yours about "reasonable reasons" why it may not be feasible.

**Reply by bluewalk** [reply]
Hi, yes I have read that post.
That’s the reason why the question arose about why the iFeel property is supported even though it’s also brand specific. It is being reported even for Samsung AC units.
